### ISSUE TITLE
Spontaneous Death/Respawn After Treatment/Basic

### ISSUE THREAD

**Issue by jacleland** [acemod/ACE3#1691]
Issue existed in 3.0.1 and 3.1.1. PID is set to yes, player is knocked out. Bandaged/epi/morphine. On rare occasions, players will suddenly respawn as if they had been killed in a matter of minutes. This occurs at anytime, most recently it happened to me after demonstrating treatment procedure to someone by taking grenade damage, enough to knock out. getDammage was returning 0.815 after I was hit. I was completely bandaged and had administered enough morphine to have clear vision. Color was good, I'm not sure how much blood I had lost (I can diag_log that out as well if necessary). I'm about ready to add a script to respawn in an attempt to evaluate the condition of the corpse, but I'm not sure if I can obtain damage related information from it (never tried). Is this expected or perhaps a bug? In basic mode, can a player still die from injuries that have been treated due to pain, blood loss, etc after being mobile for up to 10 minutes or more?

**Comment by jacleland** [comment]
Just to follow up, I've added an MP "Hit" event handler that starts a spawned func which dumps all player variables for the given unit once they have been knocked out. Once the morphine volume dropped to <0.2, I manually set the ace values for this player's blood volume to 50 to see if this caused problems. So far I've been awake and mobile for at least 20 minutes. ace pain is 0 and inpain is false.Obviously, there are many more ace_ variables associated with the player, but I don't know if any are relevant. Perhaps there's a spawned function running that will cause death/respawn under circumstances that I have not encountered during this test? Thanks in advance.
